### PR TITLE
Translation information in README.md

### PR DESCRIPTION
**Pull request by gbezyuk**
I'm dreaming of making Translation Gang an international community of open-source translators, so there's a little bit of advertisement here, not just the information on existing translation projects.

### CODE REVIEW

**Comment by chrisvfritz** [comment]
I don't mind the advertisement if it could lead to a more thriving translation community. 🙂  Can you tell me more about what you hope to do with Translation Gang? And do you want to add a contact link in that section?

**Comment by gbezyuk** [comment]
Well, there are many amazing open-source projects worldwide, and there are many awesome people ready to help with translations of these projects` documentation. But the issue is that it's enthusiasm is not a constant thing, while good project's life is long. So, there should be an organisation "in charge", taking responsibility for both completing translations and maintaining the docs in actual condition.

I seem to have succeeded in organising Translation Gang as a translators community for Russian language and Vue-related projects: while the core library docs were translated by me almost alone, at the stage of proof-reading there were already tens of people, and Nuxt.js docs were recently translated in a matter of days with only about 10% of actual translation done by me. And we also seem to be quite good at maintaining the docs in actual condition — for most of original docs updates the latency with translation is less then a day, normally just some hours.

So, Russian is not the only language which could benefit from the organised workflow, and Vue is not the only project we're interested in. Which means, we need more enthusiastic people =)

Also, there are good projects developed by people not so good at speaking English as writing the code. And these projects also could benefit from the involvement of the organised open-source translation community.

So, basically, I just want to make this world a bit better, if you could forgive me this high style =)

**Comment by gbezyuk** [comment]
And about the contact link — I've simply used the VCard link as there are no private messages at GitHub AFAIK, so I guess it's better to give an interested person a way to contact me, not just a github nickname.

**Comment by chrisvfritz** [comment]
@gbezyuk Have you added the VCard link? I'm happy to merge this after that.

**Comment by gbezyuk** [comment]
@chrisvfritz done ;)

**Comment by gbezyuk** [comment]
BTW, @kazupon could you please add information on Japanese translation?

**Comment by chrisvfritz** [comment]
Vuetiful! 😉

**Comment by kazupon** [comment]
@gbezyuk Yes, I could add information about japanese translation!
